cmd/manager/main.go

```diff
@@ -128,8 +135,27 @@ func main() {
 		os.Exit(1)
 	}
 
+	disableHTTP2 := func(config *tls.Config) {
+		setupLog.Info("disabling HTTP/2")
+		config.NextProtos = []string{"http/1.1"}
+	}
+
+	if !enableHTTP2 {
+		tlsOpts = append(tlsOpts, disableHTTP2)
+	}
+
+	metricsServerOptions := metricsserver.Options{
+		BindAddress:   metricsAddr,
+		SecureServing: secureMetrics,
+		TLSOpts:       tlsOpts,
+	}
+
+	if secureMetrics {
```

Comment:
why someone will use insecure metrics, cant we always use secureMetircs?

Reply:
It is enabled by default. The reason why it is controlled by a flag of its own is, in prod env it is recommended to setup proper certs and keys before opting in for secure metrics.

If someone does not want to do that and just wants to check something, they may disable it.

Reply:
can you document the steps to provide certificates, so that production deployments may replace automatic generated ones?

Reply:
Done